core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3121,6 +3121,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_STARTUP_TIMEOUT =
+      new Builder(Name.WORKER_STARTUP_TIMEOUT)
+          .setDefaultValue("60sec")
```

Comment:
the default value should be 600sec I guess, because there was a '10 *' at ```CommonUtils.invokeAll(callables,
          (long) callables.size() * 10 * Constants.DEFAULT_REGISTRY_GET_TIMEOUT_MS);```

Reply:
the default value of worker start time out is 10 minutes(a large worker can take a long time to register all blockers with the master in practice) and we don't want users to modify it.

Reply:
I will turn the default value to 10 mins.

Reply:
Done

Reply:
> the default value of worker start time out is 10 minutes(a large worker can take a long time to register all blockers with the master in practice) and we don't want users to modify it.

@HelloHorizon Ce said 10 mins is not good enough for the workers with large volume disks, that's why he proposed this pr to to make it configurable.  Do you have any concern for involving this new config?